packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -630,18 +582,17 @@ export class PluginLedgerConnectorQuorum
 
   public async deployContractJsonObject(
     req: DeployContractSolidityBytecodeJsonObjectV1Request,
-  ): Promise<RunTransactionResponse> {
+  ): Promise<DeployContractSolidityBytecodeV1Response> {
     const fnTag = `${this.className}#deployContractNoKeychain()`;
+    const contractJSON = req.contractJSON;
     Checks.truthy(req, `${fnTag} req`);
     if (req.contractJSON != undefined) {
-      const networkId = await this.web3.eth.net.getId();
-
       const web3SigningCredential = req.web3SigningCredential as
         | Web3SigningCredentialGethKeychainPassword
         | Web3SigningCredentialPrivateKeyHex;
       const receipt = await this.transact({
         transactionConfig: {
-          data: `0x${req.bytecode}`,
+          data: `0x${(contractJSON as any).bytecode}`,
```

Comment:
@elenaizaguirre Can this be casted to the type you specified in the openapi.json instead of any?

Reply:
Done!